admin/app/view_models/workarea/admin/reports/timeline_view_model.rb

```diff
@@ -40,13 +46,14 @@ def data_for(type)
         end
 
         def release_data
-          grouped_data.transform_values do |values|
-            (values || []).map { |v| count_releases(v.starts_at.to_date) }
+          date_range.each_with_object({}) do |date, group|
+            data = releases.select { |r| r.published_at.to_date == date }
+            group[date] = [data.count]
           end
         end
 
         def transform(data)
-          data.map { |k, v| Hash[x: k, y: v.first] }.reverse
+          data.map { |k, v| Hash[x: k.to_time.utc, y: v.first] }.reverse
```

Comment:
Does Chart.js just expect UTC?

Reply:
I can double check to see if this is still relevant. Chart.js uses Moment.js to do the conversion to local time on the fly, so that's why we were seeing the difference in dates. Since SalesByTime occurs at midnight, a `-700` offset put it one day behind in the chart.